lib/constellation.rb

```diff
@@ -0,0 +1,39 @@
+class ConstellationHttpException < StandardError
+end
```

Comment:
These are the various errors that can occur btw https://github.com/Fingertips/nap/blob/master/lib/rest/error.rb